go.mod

```diff
@@ -3,14 +3,17 @@ module github.com/filecoin-project/go-amt-ipld/v2
 go 1.12
 
 require (
-	github.com/ipfs/go-block-format v0.0.2
+	github.com/filecoin-project/go-amt-ipld v1.0.0
 	github.com/ipfs/go-cid v0.0.3
-	github.com/ipfs/go-datastore v0.1.0
-	github.com/ipfs/go-ipfs-blockstore v0.1.0
+	github.com/ipfs/go-datastore v0.1.1
+	github.com/ipfs/go-ipfs-blockstore v0.1.1
 	github.com/ipfs/go-ipld-cbor v0.0.3
 	github.com/ipfs/go-log v0.0.1
-	github.com/multiformats/go-multihash v0.0.1
+	github.com/smartystreets/goconvey v1.6.4 // indirect
 	github.com/stretchr/testify v1.3.0
-	github.com/whyrusleeping/cbor-gen v0.0.0-20190917003517-d78d67427694
-	golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
+	github.com/warpfork/go-wish v0.0.0-20200122115046-b9ea61034e4a // indirect
```

Comment:
Commentary: it would be very valuable to scope down the dependencies of go-ipld-cbor to avoid large amounts of code being pulled into things like Filecoin.